src/poliastro/constants/mean_elements.py

```diff
@@ -108,7 +108,7 @@
 
 mean_a_uranus = Constant(
     "mean_a_uranus",
-    "Saturn's orbit mean semimajor axis around Sun",
+    "Uranus's orbit mean semimajor axis around Sun",
```

Comment:
Good catch, @DhruvJ22. No need to add an extra `s` though:

```suggestion
    "Uranus' orbit mean semimajor axis around Sun",
```

Reply:
@jorgepiloto, I think it is fine to have that extra 's' (https://www.grammarly.com/blog/possessive-case/) since Uranus is a singular noun. Do you have any suggestions on how to get around the quality error?

Reply:
Also, do you think the tests in the test_cr3bp_char_quant.py are sufficient tests, or do I need any additional tests?